### DISCUSSION TITLE
when the program starts the interface window in mac os is very small

### DISCUSSION THREAD

**Discussion by tano70** [mpv-player/mpv#15220] [Q&A] [answered]
when the program starts the video window is very small! is there a way to increase the size of the window? maybe with a setting to put in the configuration file? with the old version the window when the program starts was very large, now it is very small. I use macbook pro m3 max with osx 14.7.1

**Comment by Akemi** [comment, marked as answer]
see https://github.com/mpv-player/mpv/blob/master/DOCS/interface-changes.rst

```
- change `--hidpi-window-scale` default to `no`
```

set `hidpi-window-scale=yes` in your config to get the old behaviour back.

**Reply by tano70** [reply]
> hidpi-window-scale=yes

thanks! I'll try it right away
works great!! thanks again

**Comment by tano70** [comment]
since you are so kind I take this opportunity to ask you if you can help me solve another little problem, when I put the application in full screen (F key or from the menu), an annoying popup appears that says "game mode active" is there some setting to put in the configuration to avoid this useless message?

**Reply by Akemi** [reply]
short answer, sadly no. please report it to Apple so it can be disabled in the Notification Settings.

longer answer, you can modify the App Bundle's `info.plist` file and remove the game category (https://github.com/mpv-player/mpv/pull/13558/files).

see also https://github.com/mpv-player/mpv/issues/14928

**Reply by tano70** [reply]
thanks!!
